### DISCUSSION TITLE
Context menu - compressed file name

### DISCUSSION THREAD

**Discussion by Scootworks** [aonez/Keka#1343] [Q&A]
Most of the time, I select multiple files, right-click them and press "compress using Keka". I defined a custom folder name, but the suggested file name is always "Archive". How can I fix that to use my defined custom name?
![shottr](https://github.com/aonez/Keka/assets/19875104/a49ffe63-875a-4692-9744-f9a3576bd2b5)

**Comment by aonez** [comment]
@Scootworks thanks for your feedback. Seems the custom name is not applying if no access to the folder is previously granted, this is something that will need to be fixed. As a workaround you can give access to that folder or any other folders that you want the custom name to be applies.

**Comment by aonez** [comment]
Already fixed it, expect an update on Tuesday 10th. Thanks again for the feedback Marc :)

**Reply by Scootworks** [reply]
awesome, thank you very much for your quick response and fix 😍
